symbols/overlay29.yml

```diff
@@ -1467,6 +1519,33 @@ overlay29:
         Accounts for situations that can prevent changing leaders, such as having stolen from a Kecleon shop. If one of those situations prevents changing leaders, prints the corresponding message to the message log.
         
         No params.
+    - name: UseSingleUseItemWrapper
+      address:
+        EU: 0x22F5C88
+        NA: 0x22F52CC
+      description: |-
+        Same as UseSingleUseItem, but the second parameter is determined automatically from monster::action_data::action_parameter[1]::action_use_idx.
+        
+        r0: User
+    - name: UseSingleUseItem
+      address:
+        EU: 0x22F5CB4
+        NA: 0x22F52F8
+      description: |-
+        Makes a monster use the specified single-use item. The item is deleted afterwards.
+        
+        r0: User (monster who used the item)
+        r1: Target (monster that consumes the item)
+    - name: UseThrowableItem
+      address:
+        EU: 0x22F5E78
+        NA: 0x22F54BC
+      description: |-
+        Makes a monster use the specified throwable item.
+        
+        If the item's category is CATEGORY_THROWN_LINE or CATEGORY_THROWN_ARC, the game will attempt to decrement the count of the used item by 1. If it's not or there's only 1 item left, it is destroyed instead.
+        
+        r0: User (monster who used the item)
```

Comment:
With these three functions, Is the actual item to be used stored on the user struct somewhere? Could use clarification what "specified" means.

Reply:
I'll clarify it.